rotkehlchen/tests/unit/accounting/test_default_settings.py

```diff
@@ -170,7 +190,7 @@ def test_accounting_receive_settings(
         ),
         cost_basis=None,
         index=0,
-        extra_data={'tx_hash': EXAMPLE_TX_HASH_HEX},
+        extra_data={'tx_hash': expected_tx_hash},
```

Comment:
something is off, shouldn't the key be `event_identifier` here if it is `history_event`? Mentioning due to `_gain_one_ether`

Reply:
Yeah, you can see above that there is a condition
```python
if entry_type == 'history_event':
        expected_tx_hash = f'rotki_events_{EXAMPLE_TX_HASH_HEX}'
    else:  # entry_type == 'evm_event'
        expected_tx_hash = EXAMPLE_TX_HASH_HEX
```

Reply:
Is related to the other comment, I meant that the key shouldn't be `tx_hash`